backend/data/road_segmenting/nvdbapi.py

```diff
@@ -0,0 +1,756 @@
+#All credit should go to the creator of this file
+#https://github.com/LtGlahn/nvdbapi-V2/blob/master/nvdbapi.py
+# -*- coding: utf-8 -*-
+
+import six  # python 2 vs 3 compability library
+import json
+import requests
+from warnings import warn
+import os
+from copy import deepcopy
+
+# import pdb
+
+# Uncomment to silent those unverified https-request warnings
+requests.packages.urllib3.disable_warnings()
+
+"""Bibliotek for å hente data fra NVDB api V2 (og senere versjoner)
+Har 2 klasser, nvdbVegnett og nvdbFagdata
+"""
+
+
+class nvdbVegnett:
+    """Klasse for spørringer mot NVDB for å hente vegnett.
```

Comment:
Same as the other file, consider translating comments. Consider translating code too, to be honest. It's really awkward to have norwegian class names etc. Not that hard with PyCharm, but I get that it can feel like a waste of time.